config/manager/manager.yaml

```diff
@@ -44,6 +44,8 @@ spec:
           value: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
         - name: GOVERNANCE_POLICY_FRAMEWORK_ADDON_IMAGE
           value: quay.io/open-cluster-management/governance-policy-framework-addon:latest
+        - name: HOSTING_CLUSTER_NAME
```

Comment:
How do you expect this to get populated? The hosting cluster can be different for different hosted clusters.

Reply:
This env is for our kind test so the default is` local-cluser`

Reply:
so in Prod this going to be empty